jerry-core/ecma/builtin-objects/ecma-builtin-number.inc.h

```diff
@@ -39,6 +39,11 @@ NUMBER_VALUE (LIT_MAGIC_STRING_MAX_VALUE_U,
               ECMA_BUILTIN_NUMBER_MAX,
               ECMA_PROPERTY_FIXED)
 
+/* ECMA-262 v6.0, 20.1.2.6 */
+NUMBER_VALUE (LIT_MAGIC_STRING_MAX_SAFE_INTEGER_U,
+              ECMA_BUILTIN_NUMBER_MAX_SAFE_INTEGER,
+              ECMA_PROPERTY_FIXED)
```

Comment:
As `Number.MAX_SAFE_INTEGER` is an ES6/ES2015 addition, all these code paths should be guarded by a config macro, most probably by `#ifndef CONFIG_DISABLE_ES2015_BUILTIN`.